src/Handlers/ShortcodeParametersTransformHandler.php

```diff
@@ -161,8 +183,12 @@ protected function _getService($serviceId)
                 $b->lit($serviceId)
             )
         );
+
         $services = $this->serviceSelectResourceModel->select($condition);
 
-        return isset($services[0]) ? $this->_normalizeIterable($this->serviceTransformer->transform($services[0])) : null;
+        reset($services);
```

Comment:
AFAIK, `reset()` returns the first element after reset. No need for `current()` - it was there just for demonstration.